src/app/components/Comment.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+
+export default class User extends Component {
+  render() {
+    const { postid, id, name, email, body } = this.props;
+    return (
+      <div className="card border-secondary mb-3">
+        <div className="card-header">
+          <Link to={`/comments/${this.props.id}`}>{name}</Link>
```

Comment:
```suggestion
          <Link to={`/comments/${id}`}>{name}</Link>
```
раз уж достали `id`)